wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+# osu!(lazer) feature comparison
+
+*For a more condensed list of features, see [Help centre/Upgrading to lazer § Feature comparison](/wiki/Help_centre/Upgrading_to_lazer#feature-comparison)*
+
+This page documents all new and missing features compared to osu!(stable), as well as differences affecting gameplay.
+
+## New features
+
+- Cross-platform support
+  - Native Windows 8.1+, macOS, Linux, iOS, and Android support, 
+  - Mobile input methods for all game modes
+- Input
+  - Built-in tablet driver
+  - Midi controller support
+  - Gamepad support
+- Redesigned interface
+  - First-run setup wizard
+  - Storyboards in the main menu
+  - Rewind F2
+  - Hiding difficulties
+  - Soft deletion
+  - More screen scaling options
+  - Settings can be changed everywhere and they'll take effect immediately
+  - New statistics in the results and song select screens
+- Website integration for leaderboards, news, wiki, changelog, beatmap listing, beatmap info, and profile pages
+- Lots of new mods
+- Mod customisation options
+- New multiplayer features
+  - New "playlist" feature
+  - Unlimited lobby sizes
+  - Automatic beatmap queues
+- New editor features
+  - Per-segment curve types for sliders
+  - Slider splitting and merging
+  - New "verify" tab
+- Support for custom game modes
+- Gameplay
+  - Two new built-in skins
+  - Replay seeking
+  - Beatmap (local) offset auto-calibration
+- Built-in skin editor with customisable skinning components 
+  - Song density graph
+  - Performance points counter
+  - Unstable rate counter
+  - Longest combo counter
+  - Custom text components, beatmap information displays
+  - Custom static sprites
+  - Judgement counter
+  - Clicks per second counter
+- Snaking out sliders in the osu! mode
+- No black bar behind the playfield in osu!taiko
+- Timing-based note colouring in osu!mania
+
+<!-- - osu!catch ?-->
+
+## Gameplay differences 
+
+<!--
+- Multiplayer
+  - TAG co-op mode doesn't exist
+-->
+
+### General
+
+#### Song playback ramps up and down when pausing and unpausing
+
+This makes it harder to continue from a pause in the middle of active gameplay, preventing abuse of pausing to some extent.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### The Flashlight gradient does not match stable
+
+<!-- comparison images for relevant game modes  -->
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### The Easy mod no longer pauses gameplay when recovering from failure
+
+<!-- paragraph? -->
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | No |
```

Comment:
no, just not implemented yet i guess

Reply:
then i guess it would be "intentionally changed: no, needs further consideration: yes"

Reply:
Yeah, I think so.